elastic/kibana
# Collect new alerting telemetry data from Kibana index

Resolves #138996

This PR covers only the telemetry data that we can aggregate from the Kibana index (numbers 1-6 in the issue). The telemetry data from event-log index (numbers 7-9) will be covered by another follow up PR.

To Verify:

- Create some rules with tags, some without
- Use different notify when options (Every time alert is active, On status change etc)
- Snooze one of the rules
- Mute (AKA infinitive snooze) one of the rules
- Mute one of the alerts

Change telemetry task run interval to something very short time (e.g. 1 min) 
https://github.com/elastic/kibana/blob/01ecbd48f3657327cb5375b54195cd07bfc4027f/x-pack/plugins/alerting/server/usage/task.ts#L197

Expect to see below data on `https://localhost:5601/api/stats?extended=true&legacy=true`
`count_rules_by_execution_status`, 
`count_rules_with_tags`, 
`count_rules_by_notify_when`, 
`count_rules_snoozed`, 
`count_rules_muted`, 
`count_rules_with_muted_alerts`
